src/frontend/components/PlaylistPortability/index.tsx

```diff
@@ -0,0 +1,298 @@
+    mutation.mutate({
+      portable_to: playlist
+        ?.portable_to!.filter(
+          (portedToPlaylist) => portedToPlaylist.id !== oldPortabilityID,
+        )
+        .map((portedToPlaylist) => portedToPlaylist.id),
+    });
+  };
+
+  let content: JSX.Element;
+  switch (usePlaylistStatus) {
+    case 'idle':
+    case 'loading':
+      content = (
+        <Spinner size="large">
+          <FormattedMessage {...messages.loadingPlaylist} />
+        </Spinner>
+      );
+      break;
+
+    case 'error':
+      content = <ErrorMessage code="generic" />;
+      break;
+
+    case 'success':
+      content = (
+        <React.Fragment>
+          <Box align="center" direction="row" pad={{ top: 'small' }}>
+            <Text role="heading" margin="small">
+              <FormattedMessage
+                {...messages.playlistTitle}
+                values={{ title: playlist?.title, id: playlist?.id }}
+              />
+              <Button
+                aria-label={intl.formatMessage(messages.copy, {
+                  text: playlist?.id,
+                })}
+                data-clipboard-text={playlist?.id}
+                icon={<Copy />}
+                className="copy"
+              />
+            </Text>
+          </Box>
+
+          <Box width="large">
+            <PlaylistPortabilityList
+              playlist={playlist!}
+              removePlaylistPortability={removePlaylistPortability}
+            />
+            <Box>
+              <Form onSubmit={addPlaylistPortability}>
+                <FormField
+                  label={<FormattedMessage {...messages.shareWithPlaylist} />}
+                  htmlFor="port-to-playlist-uuid"
+                >
+                  <Box direction="row">
+                    <TextInput
+                      placeholder={intl.formatMessage(
+                        messages.shareWithPlaylistPlaceholder,
+                      )}
+                      id="port-to-playlist-uuid"
+                      onChange={(event) =>
+                        setNewPortabilityID(event.target.value)
+                      }
+                      value={newPortabilityID}
+                      plain
+                    />
+                    <Button
+                      type="submit"
+                      aria-label={intl.formatMessage(messages.addPortability)}
+                      icon={<AddCircle />}
+                    />
+                  </Box>
+                </FormField>
+              </Form>
+            </Box>
+
+            <Box
+              align="center"
+              margin={{ top: 'medium' }}
+              pad="small"
+              background="status-warning"
+              width="large"
+            >
+              <Text>
+                <FormattedMessage {...messages.shareWithPlaylistDetails} />
+              </Text>
+            </Box>
+          </Box>
+        </React.Fragment>
+      );
+      break;
+  }
+
+  return (
+    <Dashboard object={object}>
```

Comment:
With the `Dashboard` component modification suggested, you don't needed to render the `Dashboard` here but in the `LTIRoutes` component

```
<Route
          exact
          path={PLAYLIST_ROUTE()}
          render={() => {
            if (appData.modelName === modelName.DOCUMENTS) {
              return (
                <Dashboard object={appData.document!}>
                  <PlaylistPortability object={appData.document!} />
                </Dashboard>
              )
            }

            if (appData.modelName === modelName.VIDEOS) {
              return (
                <Dashboard object={appData.video!}>
                  <PlaylistPortability object={appData.video!} />
                </Dashboard>
              )
            }

            return <Redirect push to={FULL_SCREEN_ERROR_ROUTE('notFound')} />;
          }}
        />
```

Reply:
The `Dashboard` should not be rendered here but in the `LTIRoute`.